tests/functional/network_policy/test_network_policy.py

```
@@ -0,0 +1,453 @@
+            check_pod_connectivity(
+                source_pod_name=pod_name,
+                target_ip=svc_ip,
+                port=port,
+                namespace=ns,
+                should_succeed=False,
+                timeout=10,
+            )
+
+    @tier2
+    @pytest.mark.polarion_id("OCS-6910")
+    def test_allowed_namespace_traffic(self, network_policies_present):
```

Ruff (0.16.0)

[warning] Unused method argument: `network_policies_present`

(ARG002)